src/iinit.cpp

```diff
@@ -249,6 +249,59 @@ main( int argc, char **argv ) {
             json_string( my_env.rodsAuthScheme ) );
     }
 
+    // mz_js_20180626+
+    if ( strlen( my_env.rodsClientServerNegotiation ) == 0 ) {
+      if ( !doingEnvFileUpdate ) {
+        doingEnvFileUpdate = true;
+        printUpdateMsg();
+      }
+      printf( "Include default SSL encrypted communication [Y/n]: " );
+      std::string response = "";
+      char inbuf[100];
```

Comment:
magic number?   is the 90 related below?  can we name this length and use as a variable?  or not declare a length at all?